testdata/simple/5a75f537726934e4b62833ab6d5dca41/tests/2022-07-23T07-15-34.848Z!2022-07-23T07-15-34.869Z

```diff
@@ -0,0 +1,11 @@
+2022-07-23T07:15:34.848Z First Test Log Line
```

Comment:
We discussed over Slack that perhaps the filename could use unix timestamps. 
This would allow us to parse them with `time.Unix()`

Reply:
I remember once we discussed the filename being a base64 encoded json object.
Is that on the table?

Reply:
We could do it. What would we be base64 encoding?

Reply:
How would you feel about a fixed width unix epoch in milliseconds?

Reply:
This uses the unix epoch in ms now.